package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "open-terms-archive",
-  "version": "0.15.0",
-  "description": "Tracks and makes visible all changes to the Terms Of Service of online service providers.",
+  "version": "0.0.0",
+  "description": "Tracks and makes visible changes to the Terms of Service of online service",
```

Comment:
Is `Terms of Service` the new vocabulary to talk about service documents?

Reply:
```suggestion
  "description": "Tracks and makes visible changes to the contractual documents of online services",
```